CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ if (NOT DEFINED PETSC_ROOT)
     if (NOT (DEFINED ENV{PETSC_ROOT}))
         message(FATAL_ERROR "***** PETSC_ROOT is not set *****")
     else()
-        set(PETSC_ROOT    "$ENV{PETSC_ROOT}")
+        set(PETSC_ROOT "$ENV{PETSC_ROOT}")
```

Comment:
Please ensure that all of the relevant modifications made to the main ```CMakeLists.txt``` are made also in ```Downstream.cmake``` in order that use of the code as a library remains functional.

Reply:
Very valid point. Will have to check.

Reply:
Done. Additional verification will be appreciated. All my library usages work.